### PR TITLE
fix: wrong height for user list frame

### PR DESCRIPTION
**Pull request by asterwyx**
Height for user list frame is wrong. Define some constants and make the calculation clear.

Log: fix wrong height for user list frame
Issue: https://github.com/linuxdeepin/developer-center/issues/5616

### CODE REVIEW

**Comment by justforlxz** [comment]
/+1

File: src/session-widgets/userframelist.h [hunk 1]
```diff
@@ -24,15 +24,29 @@ DWIDGET_USE_NAMESPACE
 class UserFrameList : public QWidget
 {
     Q_OBJECT
+    Q_PROPERTY(QPair<int,int> gridBound READ gridBound WRITE setGridBound NOTIFY gridBoundChanged)
```
Comment (BLumia) [inline src/session-widgets/userframelist.h]:
```suggestion
    Q_PROPERTY(QPair<int, int> gridBound READ gridBound WRITE setGridBound NOTIFY gridBoundChanged)
```

File: src/session-widgets/userframelist.h [hunk 2]
```diff
@@ -24,15 +24,29 @@ DWIDGET_USE_NAMESPACE
...
-signals:
+    inline QPair<int, int> gridBound() const { return m_gridBound; }
+    inline int colBound() const { return m_gridBound.first; }
+    inline int rowBound() const { return m_gridBound.second; }
+
+
+Q_SIGNALS:
     void requestSwitchUser(std::shared_ptr<User> user);
     void clicked();
+    void gridBoundChanged(QPair<int,int> bound);
```
Comment (BLumia) [inline src/session-widgets/userframelist.h]:
```suggestion
    void gridBoundChanged(QPair<int, int> bound);
```

**Review by BLumia — APPROVED** [review]

**Pull request merged**